TM1py/Services/CellService.py

```diff
@@ -179,6 +179,111 @@ def execute_mdx(self, mdx, cell_properties=None, top=None):
         finally:
             self.delete_cellset(cellset_id=cellset_id)
 
+    def get_data(self, mdx, format=None, cell_properites=None, dim_properties=None, top=None):
```

Comment:
typo in cell_properties

Reply:
Maybe dim_properties should be called elem_properties?

Reply:
Maybe rename format parameter. Unnecessary use of builtin names should be avoided